compiler/src/iree/compiler/Codegen/LLVMCPU/Passes.cpp

```diff
@@ -81,6 +81,11 @@ static llvm::cl::opt<bool> clInstrumentMemoryAccesses{
                    "instrumentation is enabled."),
     llvm::cl::init(false)};
 
+static llvm::cl::opt<bool> clUseSoftmaxInterFusion(
+    "iree-use-decompose-softmax-fuse",
```

Comment:
Use the `iree-llvmcpu-` prefix for consistency with the other flags?

Reply:
Sure thing! I didn't even think about it, since this might be used at some point on the GPU side as well.
Thanks!